src/radio-statuses/observable.js

```diff
@@ -0,0 +1,23 @@
+// used in jsdoc
+// eslint-disable-next-line no-unused-vars
+import { Observable } from 'rxjs';
+import { fetchRadioStatuses } from './fetch';
+import { refreshRateInSeconds } from './refresh-rate';
+
+/**
+ * @type {Observable<RadioStatus[]>}
+ */
+
+const interval = refreshRateInSeconds * 1000;
+export const radioStatusesObservable = new Observable(function subscribe(subscriber) {
```

Comment:
```suggestion
export const radioStatusesObservable = new Observable((subscriber) => {
```

Użyłbym tutaj funkcji strzałkowej dla lepszej czytelności